### ISSUE TITLE
AD0001 in many analyzers when stylecop.json link is broken

### ISSUE THREAD

**Issue by bjornhellander** [DotNetAnalyzers/StyleCopAnalyzers#3902]
If a project consuming StyleCop.Analyzers has a broken stylecop.json link, then many analyzers fail with an ArgumentNullException.

Version: 1.2.0-beta.556

**Comment by bjornhellander** [comment]
This call probably returns null, causing problems later on:
https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/f5843ae3f419006f53cc2393872530dd602469ad/StyleCop.Analyzers/StyleCop.Analyzers/Settings/SettingsHelper.cs#L277

**Comment by jeremyregnerus** [comment]
Would this cause a case where the analyzers just stop working and require VS to be completely restarted? I'm having that issue, and I do not use stylecop.json files in the current project and wonder if by re adding this file will resolve that issue.

**Comment by bjornhellander** [comment]
@jeremyregnerus No, not as far as I understand. I suggest you create a new issue instead and describe what you are experiencing.
